src/rules.js

```diff
@@ -871,6 +871,120 @@ module.exports = [
     },
     /**
     ==================================================================
+    COLUMNS
+    ==================================================================
+    */
+    {
+        'type': 'pattern',
+        'name': 'Colm',
```

Comment:
`name` property should be an English-readable string that succinctly describes the rule, see other rules for examples